common/config/rush/pnpm-lock.yaml

```diff
@@ -3,9 +3,10 @@ dependencies:
   '@azure/arm-servicebus': 0.1.0
   '@azure/event-hubs': 1.0.8
   '@azure/ms-rest-azure-js': 1.3.3
-  '@azure/ms-rest-js': 1.8.1
```

Comment:
Before you merge, can you reset this file back to `master` and rerun `rush update` in order to update it? It's probably now out of sync with the changes made to the package.json

Reply:
Yes I noticed the conflict and am doing it now.